README.md

```diff
@@ -1,23 +1,25 @@
-# Contributte Latte - email filter
-  
-Latte filter based on Smarty idea to protect your email address.
+![](https://heatbadger.now.sh/github/readme/contributte/latte-email/?deprecated=1)
 
-[![Build Status](https://img.shields.io/travis/contributte/latte-email.svg?style=flat-square)](https://travis-ci.org/contributte/latte-email)
-[![Code coverage](https://img.shields.io/coveralls/contributte/latte-email.svg?style=flat-square)](https://coveralls.io/r/contributte/latte-email)
-[![Licence](https://img.shields.io/packagist/l/contributte/latte-email.svg?style=flat-square)](https://packagist.org/packages/contributte/latte-email)
-[![Downloads this Month](https://img.shields.io/packagist/dm/contributte/latte-email.svg?style=flat-square)](https://packagist.org/packages/contributte/latte-email)
-[![Downloads total](https://img.shields.io/packagist/dt/contributte/latte-email.svg?style=flat-square)](https://packagist.org/packages/contributte/latte-email)
-[![Latest stable](https://img.shields.io/packagist/v/contributte/latte-email.svg?style=flat-square)](https://packagist.org/packages/contributte/latte-email)
-[![PHPStan](https://img.shields.io/badge/PHPStan-enabled-brightgreen.svg?style=flat-square)](https://github.com/phpstan/phpstan)
+<p align=center>
+    <a href="https://bit.ly/ctteg"><img src="https://badgen.net/badge/support/gitter/cyan"></a>
+    <a href="https://bit.ly/cttfo"><img src="https://badgen.net/badge/support/forum/yellow"></a>
+    <a href="https://contributte.org/partners.html"><img src="https://badgen.net/badge/sponsor/donations/F96854"></a>
+</p>
 
-## Discussion / Help
+<p align=center>
+    Website 🚀 <a href="https://contributte.org">contributte.org</a> | Contact 👨🏻‍💻 <a href="https://f3l1x.io">f3l1x.io</a> | Twitter 🐦 <a href="https://twitter.com/contributte">@contributte</a>
+</p>
 
-[![Join the chat](https://img.shields.io/gitter/room/contributte/contributte.svg?style=flat-square)](http://bit.ly/ctteg)
+## Disclaimer
 
-## Documentation
+| :warning: | This project is no longer being maintained. Please use [contributte/latte](https://github.com/contributte/latte).
+|---|---|
 
-- [Setup](.docs/README.md#setup)
-- [Usage](.docs/README.md#usage)
+| Composer | [`contributte/latte`](https://packagist.org/packages/contributte/latte) |
```

Comment:
Tahle cast je spatne, melo by to referovat contributte/latte-email

Reply:
OK, opraveno